app/src/ui/repository-settings/repository-settings.tsx

```diff
@@ -288,20 +288,19 @@ export class RepositorySettings extends React.Component<
     const errors = new Array<JSX.Element | string>()
 
     if (this.state.remote && this.props.remote) {
-      if (this.state.remote.url !== this.props.remote.url) {
-        try {
-          await this.props.dispatcher.setRemoteURL(
-            this.props.repository,
-            this.props.remote.name,
-            this.state.remote.url
-          )
-        } catch (e) {
-          log.error(
-            `RepositorySettings: unable to set remote URL at ${this.props.repository.path}`,
-            e
-          )
-          errors.push(`Failed setting the remote URL: ${e}`)
-        }
+      const trimmedUrl = this.state.remote.url.trim()
```

Comment:
Almost there! We still want the second `if`:
```suggestion
      const trimmedUrl = this.state.remote.url.trim()
      if (trimmedUrl !== this.props.remote.url) {
```